src/main/java/mate/academy/spring/config/SecurityConfig.java

```diff
@@ -0,0 +1,23 @@
+package mate.academy.spring.config;
+
+import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+@EnableWebSecurity
+public class SecurityConfig extends WebSecurityConfigurerAdapter {
```

Comment:
Where is your override `configure(HttpSecurity http)` method? Add it too to configure permission for endpoints.